package.json

```diff
@@ -152,6 +158,8 @@
         "cpx": "^1.5.0",
         "css-loader": "^1.0.0",
         "cssnano": "^4.1.0",
+        "cypress": "^3.1.4",
+        "cypress-testing-library": "^2.3.5",
```

Comment:
I did not notice that we were using another library on top of cypress. I would like to see if we can reduce the need of another library for testing and stick to just official cypress stuff that comes out of the box. Like will it cause more developer friction to write integration tests if we don't use this? Or will it be a one time hit that people re-use (like 1-2 utilities)? Or is the native cypress stuff in-adequate that we need this? I don't really have a A vs B comparison to go off or any other data to make an educated guess here.

My main rationale for not wanting this is the fact that (if) we are paying for cypress then we should stick with what they offer. When I looked at the cypress documentation, I saw this

<img width="1273" alt="screen shot 2019-01-04 at 10 45 25 am" src="https://user-images.githubusercontent.com/1075325/50704930-eab72080-100d-11e9-940e-b4a215850371.png">

They don't mention that you have to additionally use another library unless I missed it. Their whole idea is that the developer isn't picking x, y, z to get stuff working. Likewise, if we pay them and need support at some point in the future, it would be easier without non official cypress stuff. In other words, we should ideally not be playing an integration engineer between two libraries. Likewise when QA/Dev want to look at documentation, they are looking in 1 place, not two.

On the other hand, if using this library makes things substantially easier, then sure...

cc @j3tan @davidjohnburrowes